docs/first-order/classify_waveforms.ipynb

```diff
@@ -0,0 +1,2327 @@
+{
```

Comment:
<blockquote>classify the difference between fast-spiking and regular-spiking neurons</blockquote><blockquote><br></blockquote><p>Try to re-read and remove unnecessary duplicated ideas."Classify the difference" does not mean much. You are either "classifying the two types of neurons" or Highlight their difference.</p><p>Rephrase.</p><p> </p><p>Also" will utilize open source data "</p><p>Try to make your verb focus on the most important thing. In this notebook, you are mostly classifying  FS and RS using Allen Institute as example data. The primary purpose is not to utilize open source data. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AllenInstitute/openscope_databook/pull/281/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>